buildSrc/src/test/groovy/org/opensearch/gradle/zipplugin/ZipPublishTest.groovy

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.gradle.zipplugin;
+
+import org.gradle.testfixtures.ProjectBuilder
+import org.gradle.api.Project
+
+
+class ZipPublishTest {
+    public void zippublish_plugin_should_add_task_to_project() {
```

Comment:
I think the convention here is `testNamesMustBeCamelCase`?

You also need to actually implement tests.

Reply:
Good point, will make this change.